src/io/flutter/preview/PreviewArea.java

```diff
@@ -218,9 +220,15 @@ private void renderWidgetOutline(@NotNull FlutterOutline outline) {
         final int y = BORDER_WITH + rect.y - rootWidgetBounds.y;
         rect = new Rectangle(x, y, rect.width, rect.height);
 
-        final JPanel widget = new JPanel();
-        widget.setBackground(pastelColors[(colorIndex++) % pastelColors.length]);
+        final JPanel widget = new JPanel(new BorderLayout());
+        final JLabel label = new JLabel(outline.getClassName());
+        label.setBorder(JBUI.Borders.empty(2, 4, 0, 0));
```

Comment:
I'm not an expert in Swing, do you know by a chance - will border expand inside the widget or outside of it? Well, maybe we actually don't care about pixel-perfect quality at all, if we don't get actual screenshots.

Reply:
I'll check - my assumption was that this would not expand the containing widget's size.